sys-process/runit/files/rc.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+run_rc_stage() {
+	local prefix="${1}"
+	local prev_opt=$(shopt -p nullglob)
```

Comment:
SC3043 (warning): In POSIX sh, 'local' is undefined.
SC3044 (warning): In POSIX sh, 'shopt' is undefined.
SC3010 (warning): In POSIX sh, [[ ]] is undefined.

If I call with /bin/sh (or bash --posix, or set -o posix for bash), these warning still work. I check the gentoo documents and didn't found specified policy on howto handle the POSIX conformance yet.

Reply:
IMO - just use `#!/bin/bash`.

Making scripts compatible with POSIX sh is a good thing in case you've no control over environment where they'll be run. In our case we do know these scripts will be used only in Gentoo Linux with bash installed (because `app-shells/bash` is included in `@system`) and available at path `/bin/bash`.

Reply:
FWIW the only potentially non-trivial bashism is `shopt`, `local` can be removed and `[[ ]]` can become `[` ]` without any consequence. Regardless it should either be `#!/bin/bash` or made POSIX, I would defer to the Gentoo devs and the maintainers choice.